components/runtime/chezscheme/chezscheme.license

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
If there's a file in the source code we can point to, that's preferable do `COMPONENT_LICENSE_FILE=...` to copying the text out of sources.